logicrunner/handle_calls.go

```diff
@@ -57,6 +57,7 @@ func (h *HandleCall) sendToNextExecutor(ctx context.Context, es *ExecutionState,
 		ObjectReference: es.Ref,
 		Parcel:          parcel,
 		Request:         request,
+		Pending:         es.pending,
```

Comment:
this is wrong. It should be like this:
`
if es.pending == Unknown {
  msg.Pending = NotInPending
} else {
  msg.Pending = es.pending
}
`